sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt

```diff
@@ -331,6 +331,7 @@ internal class AuthgearCore(
                 prompt = listOf(PromptOption.NONE),
                 loginHint = loginHint,
                 wechatRedirectURI = options?.wechatRedirectURI,
+                colorScheme = options?.colorScheme,
```

Comment:
Also support ui_locales in openUrl.

When we support ui_locales and x_color_scheme for openUrl, we need to inject them in the query. See https://github.com/authgear/authgear-sdk-flutter/pull/2/files#diff-0855c980dde3d19bfc99d65777c17622b6f977e91632f223afefb91282aeb495R275

Reply:
In case you missed this commit

Reply:
Then xamarin needs update as well. Will open issue.

Reply:
Updated, thanks. `authorizationUrl` should parse the config into queries correctly so we just need to pass ui locales here.